sktime/alignment/naive.py

```diff
@@ -82,15 +87,15 @@ def _fit(self, X, Z=None):
         return self
 
     def _get_alignment(self):
-        """Return alignment for sequences in X passed to fit.
+        """Return alignment for sequences/series passed in fit (iloc indices).
```

Comment:
Series are currently not working:

```python
from sktime.utils._testing.series import _make_series
from sktime.alignment.dtw_numba import AlignerNaive
X0 = _make_series()  
X1 = _make_series()  
d = AlignerNaive(weighted=True, derivative=True)  
align = d.fit([X0, X1]).get_alignment()  
```
is failing... Since this is a bug, I suppose I would leave this here as it is and create a separate issue.

Reply:
Issue #5363